### ISSUE TITLE
TypeError: Cannot set properties of null (setting '...'), when using markAsUsed comments

### ISSUE THREAD

**Issue by tiwariav** [atfzl/eslint-plugin-css-modules#93]
The issue is in the version 2.12.0, works fine for 2.11.4

**Comment by yuhsianw** [comment]
Hi @tiwariav , thanks for reporting this. Can you please provide the reproduce steps for the issue following this example:
### Repro Code
```javascript
import s from './animationName.scss';

export default Foo = () => (
  <div>
    <div className={s.someAnimation}></div>
  </div>
);
```
```css
@keyframes someAnimation {}
```

### ESLint Config
```json
{
  "plugins": ["css-modules"],
  "extends": ["plugin:css-modules/recommended"]
}
```

### Expected Result
Expected no errors

### Actual Result
Received error:
```
Class or exported property 'someAnimation' not found
```

**Comment by yuhsianw** [comment]
Template for above:
````markdown
### Repro Code
```javascript
import s from './animationName.scss';

export default Foo = () => (
  <div>
    <div className={s.someAnimation}></div>
  </div>
);
```
```css
@keyframes someAnimation {}
```

### ESLint Config
```json
{
  "plugins": ["css-modules"],
  "extends": ["plugin:css-modules/recommended"]
}
```

### Expected Result
Expected no errors

### Actual Result
Received error:
```
Class or exported property 'someAnimation' not found
```
````

**Comment by tiwariav** [comment]
### Repro Code
```javascript
/* eslint css-modules/no-unused-class: [2, {camelCase: true, markAsUsed:
  ['nested']
}] */
import styles from './styles.scss';

export default Foo = () => (
  <div className={styles.root}>
    <div className={styles.nested}></div>
  </div>
);
```
```css
.root {
  color: green;
  & .nested {
    color: red;
  }
}
```

### ESLint Config
```json
{
  "plugins": ["css-modules"],
  "rules": {
    "css-modules/no-undef-class": ["warn", { "camelCase": true }],
    "css-modules/no-unused-class": ["error", { "camelCase": true }],
  }
}
```

### Expected Result
Expected no errors

### Actual Result
Received error:
```
TypeError: Cannot set properties of null (setting 'nested')
```

Can check here as well.
https://codesandbox.io/p/sandbox/brave-mopsa-ycnv8f?file=%2Fsrc%2Findex.tsx%3A6%2C1

**Comment by tiwariav** [comment]
From what I have tried, its only giving error if using nested classes. Although the same works fine with previous version.
